portfolio/portfolio.go

```diff
@@ -444,3 +444,39 @@ func StartPortfolioWatcher() {
 func GetPortfolio() *Base {
 	return &Portfolio
 }
+
+// IsExchangeSupported checks if exchange is supported by portfolio address
+func IsExchangeSupported(exchange, address string) (ret bool) {
+	for x := range Portfolio.Addresses {
+		if Portfolio.Addresses[x].Address != address {
+			continue
+		}
+		exchangeList := strings.Split(Portfolio.Addresses[x].SupportedExchanges, ",")
+		if common.StringDataContainsInsensitive(exchangeList, exchange) {
```

Comment:
Looks like you could `return common.StringDataContainsInsensitive(exchangeList, exchange)`